doc/METRICS.md

```diff
@@ -1,177 +1,3 @@
 # Libp2p Metrics <!-- omit in toc -->
 
-Metrics allow you to gather run time statistics on your libp2p node.
```

Comment:
There is useful information here that isn't in the `@libp2p/prometheus-metrics` docs.

Can you please PR that repo with the information that's been removed here.

You can use the `@packageDocumentation` tag ([example](https://github.com/libp2p/js-libp2p/blob/master/src/pnet/index.ts#L1C1-L23) - [output](https://libp2p.github.io/js-libp2p/modules/pnet.html)) to get general docs to appear on a given page, and the `@example` tag to annotate individual methods, classes or interfaces with code that people can copy/paste to get started.

Reply:
Upon reconsideration I think it makes sense to keep this information here and then also add in the `@packageDocumentation` tags and examples for those who may interact with the metrics lib independently of libp2p core.